Resources/Prototypes/Catalog/Fills/Crates/materials.yml

```diff
@@ -63,6 +63,15 @@
       - id: SheetPlasma
         amount: 3
 
+- type: entity
+  id: CrateMaterialCardboard
+  parent: CrateGenericSteel
+  components:
+  - type: StorageFill
+    contents:
+      - id: MaterialCardboard
```

Comment:
```suggestion
      - id: CardboardMaterial
```

Same as above

Reply:
isn't this the entity that is put into the crate, the full cardboard stack item itself? I don't think it needs renaming no?

Reply:
Try spawning it in game, you'll see what I mean